legacy/src/main/java/org/opensearch/sql/legacy/query/OpenSearchActionFactory.java

```diff
@@ -63,16 +63,26 @@ public class OpenSearchActionFactory {
 
   public static QueryAction create(Client client, String sql)
       throws SqlParseException, SQLFeatureNotSupportedException, SQLFeatureDisabledException {
-    return create(client, new QueryActionRequest(sql, new ColumnTypeProvider(), Format.JSON));
+    return create(client, sql, false);
+  }
+
+  public static QueryAction create(Client client, String sql, boolean bypassMigrateToQueryPlan)
```

Comment:
This function only used in UT? if so, add a tag/java-doc on this function.

Reply:
i removed this in latest commit. It was an attempt to fix some test cases, but it turns out changing the test cases is easier.